app/views/partials/drawer-database.jade

```diff
@@ -37,6 +37,9 @@
         a.value(exec-topic='server user list') server user list
   h5 Database
   ul
+    li.pair(ng-if="neo4j.clusterRole")
+      .key Role in cluster:
```

Comment:
This wraps into two lines in the drawer, do you think it can be shorter so it stays on one line?